packages/uswds/src/Templates/ArrayFieldItemTemplate.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import {
+  ArrayFieldTemplateItemType,
+  FormContextType,
+  RJSFSchema,
+  StrictRJSFSchema,
+} from "@rjsf/utils";
+
+const ArrayFieldItemTemplate = <
+  T = any,
+  S extends StrictRJSFSchema = RJSFSchema,
+  F extends FormContextType = any,
+>(
+  props: ArrayFieldTemplateItemType<T, S, F>,
+) => {
```

Comment:
We prefer `export default function` over fat-arrow functions and export default later. Also copy the JSDoc from another implementation please
```suggestion
export default ArrayFieldItemTemplate<
  T = any,
  S extends StrictRJSFSchema = RJSFSchema,
  F extends FormContextType = any,
>(
  props: ArrayFieldTemplateItemType<T, S, F>,
) {
```